topics/computational-chemistry/tutorials/covid19-docking/tutorial.md

```diff
@@ -0,0 +1,342 @@
+---
+layout: tutorial_hands_on
+
+title: Virtual screening with rDock and pose scoring
+zenodo_link: ''
+questions:
+- How can candidate ligands be generated and docked to a protein?
+- How can the poses of the docked ligands be evaluated?
+objectives:
+- How to use rDock in Galaxy
+- How to perform virtual screening in Galaxy
+time_estimation: 2h  # A week ;)
+key_points:
+- Galaxy has tools and workflows for docking
+contributors:
+- simonbray
+
+---
+
+# Introduction
+{:.no_toc}
+
+This tutorial provides a companion to the work performed in March 2020 by InformaticsMatters, the Diamond Light Source, and the European Galaxy Team to perform virtual screening on candidate ligands for the SARS-CoV-2 main protease (MPro).
+
+In this tutorial, you will perform protein-ligand docking to MPro using rDock {% cite rdock %} and scoring the results using two different methods. The same tools will be used as in the original study, but with a smaller dataset.
```

Comment:
Can you also link to covid19.galaxyproject.org?

Reply:
done